### DISCUSSION TITLE
Sveltkit breaking changes

### DISCUSSION THREAD

**Discussion by JustMrMendez** [joshnuss/svelte-stripe#50] [Q&A] [answered]
Does this works after all the SveltKit changes?

**Comment by joshnuss** [comment, marked as answer]
Yes it does, because it's a client side thing.

But the docs and examples are out of date for anything backend related, like handling webhooks.

I plan to update them when SK goes 1.0. In the meantime I'm happy to answer any questions you have here, or you can always reach me on Discord

**Reply by JustMrMendez** [reply]
Perfect, my only question then is, can you process subscriptions or only one time payments? Saw an open issue from months ago. Is it till in the roadmap?

**Reply by joshnuss** [reply]
Both subscriptions and one-time payments are supported. (I believe the issue you saw was about creating an example of subscriptions)

Here's the rough idea:

1. On the server side, create the `customer` object.
2. On the server side, create the `subscription` object, attaching the `customer` and `price_id`.
3. If you set `expand: ["latest_invoice.payment_intent"]`, it'll return the payment intent.
4. Then you can use the payment intent with any components in this library.

You can see an example using node/express here:
https://github.com/stripe-samples/subscriptions-with-card-and-direct-debit/blob/main/server/node/server.js

Also note, an alternative path is using [Stripe Checkout](https://stripe.com/docs/payments/checkout) which would handle the subscription signup using Stripe's pre-build checkout.

**Comment by joshnuss** [comment]
The repo now support SvelteKit 1.0.
